modules/exploits/unix/http/schneider_electric_net55xx_encoder.rb

```diff
@@ -0,0 +1,158 @@
+﻿##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::Udp
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Auxiliary::Report
+  include Msf::Exploit::Remote::SSH
+
+  def initialize(info={})
+    super(update_info(info,
+      'Name'           => "Schneider Electric Pelco Endura NET55XX Encoder",
+      'Description'    => %q{
+        This module exploits an inadequate access control vulnerability creating a malicious
```

Comment:
This module exploits an inadequate access control vulnerability creating a malicious
        json request to the webUI encoder, thus allowing the SSH service to be enabled , and
        changing the root password. This module has been tested sucessfully on: NET5501,
        NET5501-I, NET5501-XT, NET5504, NET5500,NET5516,NET550 versions.

This description doesn't make a whole lot of sense. How about:

        This module exploits inadequate access controls within the webUI
        to enable the SSH service and change the root password.

        This module has been tested sucessfully on: NET5501, NET5501-I,
        NET5501-XT, NET5504, NET5500, NET5516, NET550 versions.

Reply:
Ok, done